README.md

```diff
@@ -60,8 +59,6 @@ The `dependencies` sub associative array can contain the following keys:
   delegator factory keys, see the
   [Expressive delegators documentation](https://docs.zendframework.com/zend-servicemanager/delegators/)
   for more details.
-- `extensions`: an associative array that maps service names to lists of
-  extension factory names, see the [the section below](#extensions).
```

Comment:
I don't think we should likely remove this feature.

We provide users a _choice_ in terms of which container implementation they wish to use, and adapt certain features we feel should be common and work the same between implementations (invokables and delegators, primarily). However, people choose container implementations based on _features_ they provide outside basic compatibility: speed, ease of configuration, extensibility, etc.

Disabling this particular feature makes it harder for end-users to configure Pimple if they want to use Pimple-specific features. Yes, this feature largely replicates what delegators do; in point of fact, however, the way we implement delegators within Pimple _is as extensions_!

Let's keep this feature, please.